index.js

```diff
@@ -177,6 +177,7 @@ function setConfig(configObject) {
     clientId: configObject && configObject.clientId || process.env.IOPIPE_TOKEN || process.env.IOPIPE_CLIENTID || '',
     debug: configObject && configObject.debug || process.env.IOPIPE_DEBUG || false,
     network_timeout: 5000,
+    timeout_millis: 50
```

Comment:
(I kid) -- I feel that's closer @coreylight but I the  `noop` part doesn't work for me in explaining what it does?